packages/playground/windows/playground/App.cpp

```diff
@@ -22,7 +22,9 @@ using namespace playground::implementation;
 /// </summary>
 App::App() {
   InitializeComponent();
+#ifndef USE_WINUI3
```

Comment:
We should start marking the dim-down features somehow. 
Here is one idea: have a FrameworkCapabilities API, which returns different things in UWP vs WinAppSDK vs. xaml islands, e.g.:

```cs
enum AppFrameworkKind {
  UWP,
  SystemXAMLIslands,
  WinAppSDK
}

enum FrameworkCapability {
  SuspendResume,
  // more
}
runtimeclass FrameworkCapabilities {
  static AppFrameworkKind AppFrameworkKind { get; }
  bool IsCapabilitySupported(Capability c);
}
```

This would then enable us to:
- autogenerate docs based on the framework capabilities
- allow native code (and JS) to query for the presence of these capabilities (e.g. like a DeviceInfo module API or similar)

Reply:
Filed #9197.